flow-nodes/api-builder-plugin-format-date/src/flow-nodes.yml

```diff
@@ -11,18 +11,28 @@ flow-nodes:
         parameters:
           date:
             name: Date
-            description: The date to be formatted.
-            required: true
+            description: The date to be formatted (e.g. "2020-08-19"). Defaults to current date/time. For more information, see https://momentjs.com/docs/#/parsing/string/
+            required: false
             initialType: string
             schema:
               type: string
           format:
             name: Format
-            description: The desired format. Supports all Moment.js supported formats.
-            required: true
+            description: The desired format. Defaults to ISO-8601 format. Supports all Moment.js formats. See more valid format examples https://momentjs.com/docs/#/parsing/special-formats/
+            required: false
             initialType: string
             schema:
               type: string
+              default: 'YYYY-MM-DDTHH:mm:ssZ'
+              description: Defaults to ISO-8601 format
+          offset:
+            name: Time zone UTC offset
+            description: The desired time zone UTC (e.g. +01:00). See Moment.js UTC offset string format in https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/09-utc-offset/
```

Comment:
I think we can call this just `UTC offset` as the name of the parameter otherwise it is a bit unwieldy. (and it's also not exactly the time zone - that's the particular region) 
`The desired UTC Offset in the format ±[hh]:[mm]` should be a sufficient description with no need for a link to moment. We can give a few additional examples if need be in the schema examples but i think the description (±[hh]:[mm]) is pretty self explanatory